test/release/examples/benchmarks/shootout/chameneosredux-fast.compopts

```diff
@@ -0,0 +1 @@
+--no-warnings
```

Comment:
Maybe this and the other .compopts for chameneous should have newlines at the end...

Reply:
Did this in PR #11010.